src/components/victory-candlestick/helper-methods.js

```diff
@@ -17,11 +17,13 @@ export default {
       const x = scale.x(datum._x1 !== undefined ? datum._x1 : datum._x);
       const y1 = scale.y(datum._high);
       const y2 = scale.y(datum._low);
+      const highWick = scale.y(datum._close);
```

Comment:
You can correct the issue you were seeing with wicks being rendered through candles if you change these to something like `highWick = scale.y(Math.max(datum._open, datum._close))`